src/App.jsx

```diff
@@ -1,44 +1,31 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [activeTabId, setActiveTabId] = useState(tabs[0].id);
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
+  const handleTabSelected = tabId => {
+    setActiveTabId(tabId);
+  };
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">
-              Tab 2
-            </a>
-          </li>
+  return (
+    <div className="section">
+      <h1 className="title">{`Selected tab is ${tabs.find(tab => tab.id === activeTabId)?.title}`}</h1>
```

Comment:
This part here works, but could be a little better. Imagine the following scenario:
You have a "tabs" array with 1000+ elements in i and the tab you want is the last one. So this whole array would be iterated to find this item, correct?
A better approach would be to store the whole "tab" object in your state instead of only it's id. So you woldn't need to iterate the array.